markov_chain.js

```diff
@@ -0,0 +1,92 @@
+'use strict'
+//Create object for Markov Chain
+class MarkovChain {
+  #transitions;
```

Comment:
Currently, the counts are being stored in transition 2d array and then the probabilities are computed. This should be done in a seperate count 2d array so that if more names are added, we can accumulate the count in the count array and update the transition matrix as required